SQL/0000-00-00-schema.sql

```diff
@@ -692,6 +698,14 @@ CREATE TABLE `mri_acquisition_dates` (
   CONSTRAINT `FK_mri_acquisition_dates_1` FOREIGN KEY (`SessionID`) REFERENCES `session` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `mri_protocol_group` (
+    `MriProtocolGroupID`   INT(4) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                 VARCHAR(255)    NOT NULL UNIQUE,
+    PRIMARY KEY (`MriProtocolGroupID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
```

Comment:
```suggestion
) ENGINE = InnoDB  DEFAULT CHARSET=utf8mb4;
```
set correctly in upgrade sql patch. need to correct all new table in 0000-00-00-schema.sql file to have utf8mb4.